targets/TARGET_NUVOTON/TARGET_NUC472/serial_api.c

```diff
@@ -1136,4 +1139,23 @@ static int serial_is_irq_en(serial_t *obj, SerialIrq irq)
 }
 
 #endif  // #if DEVICE_SERIAL_ASYNCH
+
+bool serial_can_deep_sleep(void)
```

Comment:
I think the code of this fun is the same for every target in this PR. It might be a good idea to move it to a common file.

Reply:
@fkjagodzinski `serial_can_deep_sleep` is recommended in #7698 by @c1728p9 , but it is not defined in HAL spec. I can only fix on Nuvoton targets.

Reply:
Oh, I wasn't aware of the origin of this function. This could be a useful HAL feature. :+1: 

Anyway, my original comment was only about your new code. It might be convenient not to keep a few copies of one function. In this case it would stay inside `targets/TARGET_NUVOTON/`.

Reply:
@fkjagodzinski Currently, on Nuvoton targets, HAL implementations (e.g. `serial_api.c`) for each target (**NUMAKER_PFM_NUC472**, **NUMAKER_PFM_M453**, etc.) are branched. Except these HAL implementations are resolved to merge into one, `serial_can_deep_sleep` would be kept branched for consistency.

Reply:
Sure, no problem for me.